### PR TITLE
feat: Overview screen mobile responsiveness tm-61

### PR DESCRIPTION
**Pull request by Ann-Maksay**
There's nothing to style in the header for now, as there's only an image 

- Mobile:
image 
image

- Tablets
image

- 1024px

image

-Desktop
image
image

-Since we have a maximum length of 16 characters for the first name and 26 characters for the last name.
image

image
image

### CODE REVIEW

File: apps/frontend/src/libs/components/blurred-background/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.blurred-background {
+	position: fixed;
+	inset: 0;
+	z-index: 99;
+	width: 100%;
+	height: 100%;
+	background-color: #004a40b3;
```
Comment (GEOFARL) [inline apps/frontend/src/libs/components/blurred-background/styles.module.css:7]:
```suggestion
	background-color: var(--modal-container-background);
```

File: apps/frontend/src/libs/components/blurred-background/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	backdrop-filter: blur(5px);
+	border: none;
+	border-radius: 0;
+}
+
+.blurred-background:hover {
+	background-color: #004a40b3;
```
Comment (GEOFARL) [inline apps/frontend/src/libs/components/blurred-background/styles.module.css:14]:
```suggestion
	background-color: var(--modal-container-background);
```

File: apps/frontend/src/libs/components/blurred-background/styles.module.css (hunk 1 above)
Comment (Ann-Maksay) [reply apps/frontend/src/libs/components/blurred-background/styles.module.css:7]: Це частина коду з пулл реквесту #90

Comment (ShadowOfTheSpace) [reply apps/frontend/src/libs/components/blurred-background/styles.module.css:7]: Зараз як раз виправляю)

File: apps/frontend/src/libs/components/courses/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	.list {
+		grid-template-columns: 1fr;
+	}
+}
+
+@media screen and (width <= 489px) {
+	.container {
+		padding: 20px 15px 50px;
+	}
+}
```
Comment (YourTeaChan) [inline apps/frontend/src/libs/components/courses/styles.module.css:50]: We have only 3 breakpoints: 1024px, 768px and 480px. It was mentioned in my PR https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/pull/76#discussion_r1489078279

Comment (Ann-Maksay) [reply apps/frontend/src/libs/components/courses/styles.module.css:50]: I think there needs to be confirmation from the coaches if we are really limited to just these breakpoints. I can make the design look ok on them, but it breaks between

**Review by what1s1ove — CHANGES_REQUESTED** [review]
Please, merge `main` branch into your branch

**Review by fshabanov — COMMENTED** [review]
Why is there a strange white space above the sidebar?

**Comment by Ann-Maksay** [comment]
> Why is there a strange white space above the sidebar?

This is the list of users that we temporarily render above all other content in the app component

**Comment by MappariChan** [comment]
Maybe you can add `display: none` attribute to the list above sidebar, so it will use users slice and we won't be able to see it

File: apps/frontend/src/libs/components/auth-wrapper/styles.module.css [hunk 4]
```diff
@@ -7,4 +7,17 @@
 	display: flex;
 	flex: 1;
 	flex-direction: column;
+	background-color: var(--color-background);
+}
+
+@media screen and (width <= 1025px) {
```
Comment (Vfivew) [inline apps/frontend/src/libs/components/auth-wrapper/styles.module.css:13]: You use 
(width <= 1025px)
But we used
(width <= 1024px)  in 
[here](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/blob/main/apps/frontend/src/pages/auth/styles.module.css)
Can't we use one checkpoint or is it necessary here?

**Review by Vfivew — APPROVED** [review]

File: apps/frontend/src/libs/components/auth-wrapper/styles.module.css (hunk 4 above)
Comment (Ann-Maksay) [reply apps/frontend/src/libs/components/auth-wrapper/styles.module.css]: Good point! I was mixed up and was sure our main breakpoint was 1025

File: apps/frontend/src/libs/components/auth-wrapper/styles.module.css [hunk 5]
```diff
@@ -1,10 +1,23 @@
 .page-layout {
 	display: flex;
-	height: 100vh;
+	height: 100%;
```
Comment (GEOFARL) [inline apps/frontend/src/libs/components/auth-wrapper/styles.module.css:3]:
```suggestion
	min-height: 100vh;
```
Wouldn't it be better?

Comment (Ann-Maksay) [reply apps/frontend/src/libs/components/auth-wrapper/styles.module.css:3]: We already have min-height for `body` so I think % it is fine there

File: apps/frontend/src/libs/components/app/app.tsx [hunk 6]
```diff
@@ -28,7 +28,7 @@ const App: React.FC = () => {
 
 	return (
 		<>
-			<ul>
+			<ul className="visually-hidden">
```
Comment (what1s1ove) [inline apps/frontend/src/libs/components/app/app.tsx:31]: we will remove it in a separate task

**Review by what1s1ove — APPROVED** [review]

File: apps/frontend/src/libs/components/app/app.tsx (hunk 6 above)
Comment (what1s1ove) [reply apps/frontend/src/libs/components/app/app.tsx:31]: Resolved in https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/pull/109

**Pull request merged**